exercises/ayomide_bamidele/ex02_oo_basics/src/Circle.cpp

```diff
@@ -0,0 +1,12 @@
+#include "Circle.h"
+#include <iostream>
+
+Circle::Circle(double r) : radius(r) {}
+
+double Circle::area() const { return 3.14 * radius * radius; }
```

Comment:
Member function should be comment with doxygen style comments much like free functions

Reply:
Alright.